types/wallpaper/wallpaper-tests.ts

```diff
@@ -1,7 +1,7 @@
 import wallpaper = require('wallpaper');
 
 wallpaper.set('unicorn.jpg').then(() => {});
-wallpaper.set('unicorn.jpg', {scale: 'fill'}).then(() => {});
+wallpaper.set('unicorn.jpg', { scale: 'fill' }).then(() => {});
 
 wallpaper.get().then(imagePath => {
```

Comment:
Would be great if you could add some tests for `get()` with options and for `screens()`.

Reply:
Sure!